### ISSUE TITLE
please provide hierarchy with pear style naming

### ISSUE THREAD

**Issue by cjnoyescrisp** [ApiGen/ApiGen#747]
I love apigen. One problem that we have is we have to support a large number of Zend Framework 1 projects. I know you pick up on namespaces, but can you use the _ in Parent_Class naming to extract structure.

**Comment by ek9** [comment]
Why? Can't you just use `phar` file in that case? Asking a project to switch class naming in order to support a project which is old and is past it's end-of-life phase is quite opportunistic.

**Comment by cjnoyescrisp** [comment]
Pear style naming is used by Zend Framework 1 and we have a number of
services built using it. yours is the best and want to use it. Can you
guide where modifications could be made?

**Comment by ek9** [comment]
I'm sorry I might have misunderstood your original request. Could you please clarify what you need exactly:

a) apigen to support PEAR style namespaces to generate api docs for
b) apigen to be updated so it uses PEAR style class names itself

**Comment by cjnoyescrisp** [comment]
instead of the newer php namespace code, zend framework 1, which is older
uses this kind of class naming

Crisp_Auth_Adapter

which leads to a file structure like

/Crisp/Auth/Adapter.php

and the _ in the name provides the structure

**Comment by ek9** [comment]
What is the problem with ApiGen though? Is it not able to render the API for these or?

**Comment by cjnoyescrisp** [comment]
what it renders is hard to follow

Attached is a file it generates

What I would like would be to produce a tree out of the pear named files
the way the namespace files do

**Comment by cjnoyescrisp** [comment]
[class-Zend_View_Helper_Navigation_Breadcrumbs.html.zip](https://github.com/ApiGen/ApiGen/files/703029/class-Zend_View_Helper_Navigation_Breadcrumbs.html.zip)
attached sample generated file

**Comment by TomasVotruba** [comment]
I understand what you mean.

Nowadays, even [PHPUnit](https://github.com/sebastianbergmann/phpunit/blob/e423b400100e986af2cf22ad4cdc29721fdf7828/src/Framework/TestCase.php#L11) and [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer/blob/ab6b482bc3349d3d966ce3ab351193a0defe75cc/src/Sniffs/Sniff.php#L15) are moving from PEAR to namespaced (even PSR-4) naming.

We won't support PEAR anymore. This tool already support lot of deprecated stuffs, but has no support for "new" ones, like PHP 5.5+.

I recommend you bumping to newer framework or different project, since this won't be support by more and more projects in time.
